utils.py

```diff
@@ -259,6 +258,24 @@ def get_random_choice(alist):
     return alist[index]
 
 
+def get_random_sample(alist, sample_length):
```

Comment:
I don't see why you need to break this out into utils.py. Why not just do random.sample() directly in the code?

Reply:
I did because nowhere in the code they used it directly. But now I have removed it and used it directly.